### PR TITLE
NSW-OEH pipeline to send emails to Sep (only when something is wrong)

### PR DESCRIPTION
**Pull request by mhidas**

### CODE REVIEW

**Comment by mhidas** [comment]
@lbesnard @bpasquer @ggalibert  Could one of you take a quick squiz at this please?

File: AODN/NSW-OEH/incoming_handler.sh [hunk 1]
```diff
@@ -53,7 +49,10 @@ main() {
 
     index_files_bulk $unzip_dir $dest_path $report
     if [ $? -ne 0 ]; then
+        # indexing failed... let uploader AND backup recipient know!
         notify_uploader $report "Contents of '$zipfile_basename' were extracted but publishing failed"
+        cat $report | notify_by_email $BACKUP_RECIPIENT "$subject" &&
```
Comment (bpasquer) [inline AODN/NSW-OEH/incoming_handler.sh]: i think you're missing a backslash at the end of the line

Comment (lbesnard) [inline AODN/NSW-OEH/incoming_handler.sh]: I didn't spot that !

Comment (mhidas) [inline AODN/NSW-OEH/incoming_handler.sh]: Oops, thanks. I'll have to add it in a new PR now...

**Pull request merged**